py/visdom/Makefile

```diff
@@ -0,0 +1,21 @@
+clean:
```

Comment:
I'm somewhat against adding _yet another_ scripting flow to the visdom codebase. This should either be part of the `package.json` scripts, or we should be having a discussion about the alternate we should be using instead.